core/src/main/java/org/elasticsearch/action/support/tasks/BaseTasksResponse.java

```diff
@@ -60,6 +66,17 @@ public BaseTasksResponse(List<TaskOperationFailure> taskFailures, List<? extends
         return nodeFailures;
     }
 
+    /**
+     * Rethrow task failures if there are any.
+     */
+    public void rethrowFailures(String operationName) {
+        rethrowAndSuppress(Stream.concat(
```

Comment:
I made this because it is super convenient for testing and as part of rethrottle.